Add in version string definition for role in Terraform reference (#10609)
gravitational/teleport: docs/pages/setup/reference/terraform-provider.mdx

|              `namespace`|     string |          | Resource namespace ("default" by default) |
|                `expires`|     string |          | Expiry time in RFC8829 format             |

**version**

> `version` is a not required `string` that specifies the role version (default `v3`). 

Reviewer: ```suggestion
`version` is an optional string that specifies the role version (default `v3`). 
```
